app/v1/lifecycle/systems/endpoints.py

```diff
@@ -0,0 +1,42 @@
+from fastapi import APIRouter, Path
+
+from app.data.systems import OS_DATA_MOCKED
+
+v1_router = APIRouter()
+
+
+@v1_router.get("/systems")
```

Comment:
nit: REST API endpoints should have a trailing slash. Resources can omit the trailing slash (such as a request with path parameters).

This is a [highly] [contentious] topic and that's just my opinion.

Internally, FastAPI registers both routes (with and without a trailing slash) and issues a 307 redirect to get to real endpoint.

[highly]: https://github.com/fastapi/fastapi/discussions/9328
[contentious]: https://github.com/fastapi/fastapi/pull/12145